administrator/modules/mod_sampledata/tmpl/default.php

```diff
@@ -32,7 +32,7 @@
 		<?php foreach($items as $i => $item) : ?>
 			<li class="list-group-item sampledata-<?php echo $item->name; ?>">
 				<div class="d-flex justify-content-between align-items-center">
-					<div class="sample-data__title me-2">
+					<div class="sample-data__title”>
```

Comment:
```suggestion
					<div class="sample-data__title">
```
You should be careful to use the right kind of double quotes.

Reply:
@richard67 thanks, it was a nice catch

Reply:
I only make nice catches 😛